server/svix-server/src/worker.rs

```diff
@@ -291,6 +257,117 @@ fn bytes_to_string(bytes: bytes::Bytes) -> String {
     }
 }
 
+/// Manages preparation and execution of a QueueTask type
+async fn process_task(
+    cfg: Configuration,
+    db: &DatabaseConnection,
+    cache: Cache,
+    queue_tx: &TaskQueueProducer,
+    queue_task: QueueTask,
+) -> Result<()> {
+    let msg = message::Entity::find_by_id(queue_task.clone().msg_id())
+        .one(db)
+        .await?
+        .ok_or_else(|| {
+            Error::Generic(format!(
+                "Unexpected: message doesn't exist {}",
+                queue_task.clone().msg_id()
+            ))
+        })?;
+    let payload = msg.payload.as_ref().expect("Message payload is NULL");
+
+    let msg_clone = msg.clone();
```

Comment:
Cloning msg means cloning payload. I think you probably just want to clone app_id and org_id below?